### DISCUSSION TITLE
cant login on lazer

### DISCUSSION THREAD

**Discussion by yaboiisme** [ppy/osu#15943] [Q&A] [answered]
im trying to log in on lazer but its says "forbidden" I know that the username and password are correct it just wont let me in

checked the logs and this error caught my attention, Failing request osu.Game.Online.API.Requests.SearchBeatmapSetsRequest (System.OperationCanceledException: Request cancelled)
https://drive.google.com/file/d/1o12SfkxPkyGBF8IiSbkQ9yIbDyurTsXy/view?usp=sharing
[network.log](https://github.com/ppy/osu/files/7655514/network.log)

**Comment by bdach** [comment, marked as answer]
please post the full logs. the error you mention is not likely to be related.

also please make sure you are using your username and not your email to log in.

**Reply by yaboiisme** [reply]
1. They are posted. 2.i am using my username

**Reply by bdach** [reply]
not all of them, just the network one. i'd be interested in looking at `runtime.log` too.

**Reply by yaboiisme** [reply]
[runtime.log](https://github.com/ppy/osu/files/7656008/runtime.log)
[updater.log](https://github.com/ppy/osu/files/7656009/updater.log)
[database.log](https://github.com/ppy/osu/files/7656010/database.log)
[network.log](https://github.com/ppy/osu/files/7656011/network.log)
[performance.log](https://github.com/ppy/osu/files/7656012/performance.log)

**Reply by yaboiisme** [reply]
runtime shouldnt be too long as I only started using lazer today

**Reply by peppy** [reply]
The network log is missing any kind of authentication request.

Could you please try logging in and then provide the `network.log` after doing so? Note that the logs are reset on each game startup.

**Reply by yaboiisme** [reply]
nevermind, the problem fixed itself when I tried again today
